ssg/templates.py

```diff
@@ -64,6 +67,16 @@ def preprocess(self, parameters, lang):
             uppercases[k.upper()] = v
         return uppercases
 
+    def looks_like_template(self):
+        if not os.path.isdir(self.template_root_directory):
+            return False
+        if os.path.islink(self.template_root_directory):
+            return False
+        template_sources = glob.glob(os.path.join(self.template_path, "*.template"))
+        if not os.path.isfile(self.template_yaml_path) and not template_sources:
+            return False
+        return True
+
```

Comment:
I think this check could be done during `Template` instantiation, sot that it every instance is checked.

Reply:
This is not a big deal, merging to unblock CI.

Reply:
I was thinking of it, but the constructor can't return any value - it just initializes the object, and we want to prevent its addition to the templates dictionary.
Then, I was thinking of a static method that one would use before instantiating `Template`, but that one would have to calculate all paths again, and that would be either duplication, or it would require even more code extractions.

Reply:
Yeap, I came to similar conclusion.
Ideally all instances would come valid and wouldn't need one to remember to call a function to check it.
This may warrant a comment or documentation note...